### ISSUE TITLE
Add debug for invalid GameProfiles on skull blocks/items patch no longer functions

### ISSUE THREAD

**Issue by lynxplay** [PaperMC/Paper#7983]
### Expected behavior

The patch https://github.com/PaperMC/Paper/blob/0bdbcd9d56e0b6e55274486c3162f37f59c41d6d/patches/removed/1.19/0823-Add-debug-for-invalid-GameProfiles-on-skull-blocks-i.patch was introduced to make debugging invalid skull block entities or items easier to locate.

It is expected that the provided debug flag is usable to enable the debug message.

### Observed/Actual behavior

With 1.19, authlib was updated to 3.5.41. In either that version (or an intermediate one between 1.18.2's 3.3.39 the `com.mojang.authlib.yggdrasil.YggdrasilMinecraftSessionService` wraps the logic in `YggdrasilMinecraftSessionService#fillGameProfile(GameProfile,boolean)` in a `try-catch`.

While this was always the case, the try catch was expanded to now also catch any `IllegalArgumentException`, which is the exception throw if an invalid game profile is constructed.

As such, the patch introduced `try-catch` no longer catches anything, as the exception is already handles by the library and logged to the console without any debug information.

### Steps/models to reproduce

Have an invalid skull in your world. 
A user reported this on the discord: https://discord.com/channels/289587909051416579/289587909051416579/986345805013209158

### Plugin and Datapack List

N/A

### Paper version

Flawed logic exists on 48f7a1a (current latest).

**Comment by HighPriestKing** [comment]
What config file do i edit to add this patch?

**Comment by electronicboy** [comment]
the patch exists in the server with or without config tweaks, there is a system property to debug invalid skull data, but, that no longer works as the patch is borked due to some mojang authlib changes, of which you'd need to modify authlib or the patch itself to fix.

**Comment by HighPriestKing** [comment]
where in the file directory can i find the authlib file to edit ?

**Comment by electronicboy** [comment]
you'd need to import it, as per the contrib guide.

**Comment by HighPriestKing** [comment]
I'm sorry, I'm only 1 year into dev work on minecraft servers.. and this is new to me! Could you provide a link to the guide on how to go about handling this? >.< or is there a youtube video someone threw up by anychance?

**Comment by HighPriestKing** [comment]
Still no clear direction on how to add a "patch" to my server... Please help!!

**Comment by sovtec** [comment]
Have same issues.

**Comment by electronicboy** [comment]
This patch is broken, it already exists within the server, but it does not
perform the functionality it should do due to changes elsewhere. You do not
need to add a patch for this issue, somebody needs to make changes
elsewhere.

**Comment by HighPriestKing** [comment]
somebody as in majong?...

**Comment by electronicboy** [comment]
No, it's not mojangs changes, it's ours.

**Comment by HighPriestKing** [comment]
ohhhhhh. So im guessing someone is working on it? and i should stop worrying about it? lol..

**Comment by HighPriestKing** [comment]
is that a yes? We just wait for someone in paper to fix it?... and is there a way to disable payer heads from dropping in the meantime?

**Comment by electronicboy** [comment]
It's in the todo list, this is not a paper issue, however, this is something on your server creating invalid skulls, this is just a tool to aid in picking up what caused it

**Comment by HighPriestKing** [comment]
okay soooooo we updated to 1.19 and the issue still occurs. :/

Any updates on this?.

**Comment by electronicboy** [comment]
No, I started working on the patch and wasn't feeling well, dumped it somewhere for somebody to fix it up

**Comment by HighPriestKing** [comment]
okay. Hope you're got better!! ^_^
